src/project-tests/scatter-plot-tests.js

```diff
@@ -315,6 +319,16 @@ export default function createScatterPlotTests() {
           document.getElementById('legend'),
           'There should be an element with id="legend" '
         );
+        var legendText;
+        if (document.querySelector('#legend text') !== null) {
```

Comment:
Also, I just noticed that you are testing for text inside the legend. Could the test text reflect that? Maybe this is overkill, but would "'13. I can see a legend containing descriptive text that has id="legend".' be clearer and more accurately reflect the tests you are making?